documentation/installation.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: kimai
-title: Documentation for Kimai
-description: Documentation for Kimai Time-Tracking
+title: Kimai Installation
+description: Kimai Installation
```

Comment:
This is going to be shown in the google search results. Could you re-phrase that a little bit ... so make it at least 100 character longs?

Reply:
oh, I'm not good in SEO. Need to do some research on how to optimize the description

Reply:
160 character is the best. Just think about the small snippet, that is displayed under link the search result link. What would yourself when you see the result? Well, it wasn't better before ;-) but we could make it even more shiny